scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla/balrog:dockerhub"
+repo_token=$(curl ${password_url} | python -c 'import json, sys; a = json.load(sys.stdin); print a["secret"]["repo_token"]')
```

Comment:
Oh! I get. So if the test isn't run on taskcluster, it won't send coverage at all.